CODE/functions/simulated_bin_summary.R

```diff
@@ -0,0 +1,56 @@
+
+simulated_bin_summary <- function(dates, scores, pos) {

```

Comment:
I agree that it is cleaner passing in vectors than the entire data.table.

I'd use `positives` than `pos` for the variable name. Otherwise, it's possible to think the variable means position in the schedule.